lib/src/widgets/chat_groupedlist_widget.dart

```diff
@@ -234,33 +202,27 @@ class _ChatGroupedListWidgetState extends State<ChatGroupedListWidget>
   Future<void> _onReplyTap(String id, List<Message>? messages) async {
     // Finds the replied message if exists
     final repliedMessages = messages?.firstWhere((message) => id == message.id);
-
+    final repliedMsgAutoScrollConfig =
+        chatListConfig.repliedMessageConfig?.repliedMsgAutoScrollConfig;
+    final highlightDuration = repliedMsgAutoScrollConfig?.highlightDuration ??
+        const Duration(milliseconds: 300);
     // Scrolls to replied message and highlights
     if (repliedMessages != null && repliedMessages.key.currentState != null) {
       await Scrollable.ensureVisible(
         repliedMessages.key.currentState!.context,
         // This value will make widget to be in center when auto scrolled.
         alignment: 0.5,
-        curve: widget.repliedMessageConfig?.repliedMsgAutoScrollConfig
-                .highlightScrollCurve ??
-            Curves.easeIn,
-        duration: widget.repliedMessageConfig?.repliedMsgAutoScrollConfig
-                .highlightDuration ??
+        curve:
+            repliedMsgAutoScrollConfig?.highlightScrollCurve ?? Curves.easeIn,
+        duration: repliedMsgAutoScrollConfig?.highlightDuration ??
             const Duration(milliseconds: 300),
```

Comment:
Use variable created before condition `highlightDuration`
```suggestion
        duration: highlightDuration,
```